src/wp-includes/functions.php

```diff
@@ -1904,7 +1904,8 @@ function wp_nonce_field( $action = -1, $name = '_wpnonce', $referer = true, $ech
  * @return string Referer field HTML markup.
  */
 function wp_referer_field( $echo = true ) {
-	$referer_field = '<input type="hidden" name="_wp_http_referer" value="' . esc_attr( wp_unslash( $_SERVER['REQUEST_URI'] ) ) . '" />';
+	$request_url   = remove_query_arg( '_wp_http_referer' ) );
```

Comment:
Remove extra `)` that fixed the unit test errors.
```suggestion
	$request_url   = remove_query_arg( '_wp_http_referer' );
```

Reply:
agreed with @mukeshpanchal27.
need to remove extra closing bracket which causing error.